### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by silverdaz**

### CODE REVIEW

File: docs/inbox.rst [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+The ``account`` *type* of the PAM module is a pass-through. It
+succeeds.
+
+"Refreshing" the last access time is done by the ``setcred``
+service. The latter is usually called before a session is open, and
+after a session is closed. Since we are in a chrooted environment when
+the session closes, ``setcred`` is bound to fail. However, it
+succeeded on the original login, and it will again on the subsequent
+logins. That way, if a user logs again, within a cache TTL delay, we
```
Comment (viklund) [inline docs/inbox.rst]: if a user log _in_ again

File: docs/index.rst [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+The Local EGA project is divided into several microservices.
+
++-----------+--------------------------------------------------------------------------------------------------+
+| Service   | Description                                                                                      |
++===========+==================================================================================================+
+| db        | A Postgres database with appropriate schema                                                      |
++-----------+--------------------------------------------------------------------------------------------------+
+| mq        | A RabbitMQ message broker with appropriate accounts, exchanges, queues and bindings              |
++-----------+--------------------------------------------------------------------------------------------------+
+| inbox     | SFTP server, acting as a dropbox, where user credentials are in the db component                 |
```
Comment (viklund) [inline docs/index.rst]: They are not in the db anymore right.

File: docs/index.rst [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+================
+NBIS - Local EGA
+================
+
+The Local EGA project is divided into several microservices.
+
++-----------+--------------------------------------------------------------------------------------------------+
+| Service   | Description                                                                                      |
++===========+==================================================================================================+
+| db        | A Postgres database with appropriate schema                                                      |
```
Comment (viklund) [inline docs/index.rst]: Maybe say what it stores instead of just "appropriate schema".

File: docs/index.rst [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
++-----------+--------------------------------------------------------------------------------------------------+
+| mq        | A RabbitMQ message broker with appropriate accounts, exchanges, queues and bindings              |
```
Comment (viklund) [inline docs/index.rst]: Maybe something about that this what connects all the services together.

File: docs/index.rst [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+``CEGA``, or ``CentralEGA``, and *any* Local EGA instance as ``LEGA``,
+or ``LocalEGA``.  When two or more Local EGA instances are involved,
+we will use ``LEGA<i>`` for Local EGA instance ``<i>``.
+
+================
+NBIS - Local EGA
+================
+
+The Local EGA project is divided into several microservices.
+
```
Comment (viklund) [inline docs/index.rst]: I think you should add a column to this table that is status, with a couple of states. for example:

To Be Done, Currently implementing, Testing, Production Ready.

This was one of the main things that niclas wanted out of this if I remember correctly.

Comment (viklund) [reply docs/index.rst]: The column header could be "Status (as of 2018-01-16)"

File: CONTRIBUTING.rst [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+   
+   - should use ``Estimate=1``,
+   - should be connected to:
+
+     * an ``Epic``,
+     * a ``Milestone`` and
+     * a ``User story``
+     * ... or several.
+
+   Do :bolditalic:`not` ask us to merge it into ``master``. We will use the ``dev`` branch.
```
[the remark this answers is not in the document]
Comment (silverdaz) [reply CONTRIBUTING.rst]: ok.

How about that way?

N.B: Pull requests are done to the dev branch. Pull requests to master are rejected.

File: CONTRIBUTING.rst [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+
+#. It is possible that your PR requires changes (because it creates
+   conflicts, doesn't pass the integrated tests or because some parts
+   should be rewritten in a cleaner manner, or because it does not
+   follow the standards, or you're requesting the wrong branch to pull
+   your code, etc...) In that case, a reviewer will request changes
+   and describe them in the comment section of the PR.
+
+   You then update your branch with new commits and ping the reviewer
+   on the slack channel. (Yes, we respond better there).
```
[the remark this answers is not in the document]
Comment (silverdaz) [reply CONTRIBUTING.rst]: ok

File: docs/index.rst (hunk 5 above)
Comment (silverdaz) [reply docs/index.rst]: Ok. On it.

Comment (silverdaz) [reply docs/index.rst]: Ready.

Fontawesome does not load on the ReadTheDocs site at the moment, but they will fix it. So the icons don't show up.

**Comment by silverdaz** [comment]
Ready.
I have included the comments now.
I think I went through all of them.
I hope I didn't miss any.

**Pull request merged**